src/meson-wrapper/conf/mxe-crossfile-internal.meson.in

```diff
@@ -0,0 +1,8 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+# Do not use this file outside of MXE's build system!
+# It is intended to allow using ccache.
+
+[binaries]
+c = '@PREFIX@/@BUILD@/bin/@TARGET@-gcc'
+cpp = '@PREFIX@/@BUILD@/bin/@TARGET@-g++'
```

Comment:
I'm seeing errors like:
```
Running "/usr/lib/mxe/usr/x86_64-pc-linux-gnu/bin/i686-w64-mingw32.static-gcc --version" gave "[Errno 2] No such file or directory: '/usr/lib/mxe/usr/x86_64-pc-linux-gnu/bin/i686-w64-mingw32.static-gcc'"
```
and I think it's caused by not enabling ccache.

Reply:
Enabling ccache lets the build proceed.

Reply:
I thought ccache was working already with meson builds. What is the solution? Your proposal to fix?

Reply:
Yes, it seems to work with ccache, but not without it. The `@PREFIX@/@BUILD@/bin/@TARGET@-gcc` links only exist when ccache is enabled.

I'm not sure how best to handle it with meson - for cmake, [we remove the ccache.cmake config file](https://github.com/mxe/mxe/blob/master/src/ccache.mk#L92).

Maybe something like (untested):
```
MXE_MESON_OPTS = \
    --buildtype=release \
ifeq (mxe,$(MXE_USE_CCACHE))
    --cross-file='$(PREFIX)/$(TARGET)/share/meson/mxe-crossfile-internal.meson'
endif
```

Reply:
This should work:
```diff
diff --git a/Makefile b/Makefile
index 06a34e17..286d2882 100644
--- a/Makefile
+++ b/Makefile
@@ -142,7 +142,8 @@ MXE_NINJA = '$(PREFIX)/$(BUILD)/bin/ninja'
 # unless your changes only apply to building MXE's packages
 MXE_MESON_OPTS = \
     --buildtype=release \
-    --cross-file='$(PREFIX)/$(TARGET)/share/meson/mxe-crossfile-internal.meson'
+    $(if $(findstring mxe,$(MXE_USE_CCACHE)), \
+    --cross-file='$(PREFIX)/$(TARGET)/share/meson/mxe-crossfile-internal.meson')

 PKG_CONFIGURE_OPTS = \
     $(_$(PKG)_CONFIGURE_OPTS) \
```

Reply:
> Amazing, a fairly simple fix (untested on Linux) - I think I'm going to like meson:
> 
> ```diff
> diff --git a/src/glib.mk b/src/glib.mk
> index a099e423..db09f764 100644
> --- a/src/glib.mk
> +++ b/src/glib.mk
> @@ -22,6 +22,9 @@ endef
> 
>  define $(PKG)_BUILD_$(BUILD)
>      # native build
> +    $(if $(findstring darwin, $(BUILD)), \
> +        CPPFLAGS='-I$(PREFIX)/$(TARGET).gnu/include' \
> +        LDFLAGS='-L$(PREFIX)/$(TARGET).gnu/lib' \)
>      '$(MXE_MESON_NATIVE_WRAPPER)' \
>          --buildtype=release \
>          -Dtests=false \
> ```

Applied.

Reply:
> ```diff
> +    $(if $(findstring mxe,$(MXE_USE_CCACHE)), \
> +    --cross-file='$(PREFIX)/$(TARGET)/share/meson/mxe-crossfile-internal.meson')
> ```

Also applied